javascript/code-challenges/challenges-05.test.js

```diff
@@ -0,0 +1,401 @@
+"use strict";
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function that iterates over an array of people objects
+and creates a new list of each person's full name using the array method 'map'.
+Each object will have the shape {firstName:string, lastName:string}
+E.g. [ { firstName:"Jane", lastName:"Doe" }, { firstName:"James", lastName:"Bond"}]
+should convert to ["Jane Doe", "James Bond"]
+Note the space in between first and last names.
+You can assume that neither firstName nor lastName will be blank
+------------------------------------------------------------------------------------------------ */
+const toLastNames = (people) => {
+  return people.map((obj) => obj.firstName + " " + obj.lastName);
+};
```

Comment:
Nice work!
I prefer template literals, as they make it easier to read, for me at least.
```js
return people.map(obj => `${obj.firstName} ${obj.lastName}`);
```
But this is entirely personal preference.